inference-engine/tests/functional/plugin/conformance/test_runner/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME conformanceTests)
+
+list(APPEND EXPORT_DEPENDENCIES
+        gflags
+        funcSharedTests
+    )
+
+addIeTargetTest(
+        NAME ${TARGET_NAME}
```

Comment:
Why do we need to have a separate target if it is based on plugin functional tests?
Can we just add a specific mode to run pluginFuncTests?

Reply:
This target should be separated from other functional tests because conformance suit have to be the same for all plug-ins and mustn't be configurable by plug-ins.

Reply:
I agree that it shouldn't be a separate target, just a separate executable. 
Such IR runner may be useful not only as part of the conformance suite but also as a development\debugging tool to run arbitrary IR to explore inference results.

Reply:
Done